File: Programmers/Lv.0/OX퀴즈/120907.js
function solution(quiz) {
    const answer = [];
    
    for (let i=0; i<quiz.length; i++) {
        const [X, operator, Y, equal, Z] = quiz[i].split(' ');
        const sign = (operator === '+') ? 1 : -1;
        answer[i] = (Number(X) + (sign * Number(Y)) === Number(Z)) ? 'O' : 'X';
    }
    
    return answer;
}

File: Programmers/Lv.0/OX퀴즈/120907.java
class Solution {
    public String[] solution(String[] quiz) {
        String[] answer = new String[quiz.length];
        
        for (int i=0; i<quiz.length; i++) {
            String[] arr = quiz[i].split(" ");
            
            String operator = arr[1];
            int X = Integer.parseInt(arr[0]);
            int Y = Integer.parseInt(arr[2]);
            int Z = Integer.parseInt(arr[4]);
            
            int sign = (operator.equals("+")) ? 1 : -1;
            answer[i] = (X + (sign * Y) == Z) ? "O" : "X";
        }
        
        return answer;
    }
}
